src/shared/modules/dbMeta/dbMetaDuck.ts

```diff
@@ -128,6 +136,23 @@ export const getActiveDbName = (state: any) =>
  * Helpers
  */
 
+export const VERSION_FOR_EDITOR_HISTORY_SETTING = '4.3.0'
+
+export const versionHasEditorHistorySetting = (version: string) =>
+  semver.gte(version, VERSION_FOR_EDITOR_HISTORY_SETTING)
+
+export const supportsEditorHistorySetting = (state: any) =>
+  isEnterprise(state) && versionHasEditorHistorySetting(getVersion(state))
+
+export const shouldAllowOutgoingConnections = (state: any) =>
+  !isEnterprise(state) || getAllowOutgoingConnections(state)
```

Comment:
I think this would send outgoing connections before we connect to a database? I think that's a no-go in airgapped deployments